src/lib/components/ProjectPage/projectTabs/ProjectAccounts/ProjectAccounts.consts.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  EFilterItemType,
+  IFilterItem,
+} from '@/lib/components/commons/FilterPanel';
+import { dateFilterSchema } from '@/lib/components/commons/FilterPanel/FilterPanel.consts';
+import { EProjectAccountsFields } from '@/lib/consts/accounts';
+import { EProjectFields } from '@/lib/consts/projects';
+import { z } from 'zod';
+
+export const projectAccountsFilterSchema = z.object({
+  // [EProjectFields.Status]: z.object({
```

Comment:
why this code is commented? if it's by purpose , just remove it

Reply:
in case you save a block of code for later use , you need to add a comment that explain why the code is commented...
e.g. :
```
//TODO: decide later if we need those filters , (currently comment them out):
```